index.js

```diff
@@ -52,7 +54,7 @@ async function start() {
 
         db.setState(prices);
     } catch (e) {
-        console.error('There was a problem updating the values');
+        console.error('There was a problem updating the values', e);
```

Comment:
What do you think about using the message attribute instead of all the error object?